### DISCUSSION TITLE
184

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#155] [General]
# 潮流周刊第184期 - mamala

封面图拍摄来周末去杭州远洋乐堤港吃了顿 mamala 西餐，相比之前去人要少一些，不知道是天气实在太热的缘故还是啥，这家的二楼窗外和室内风景很好，西餐很好吃，挺适合仪式感日子。

https://weekly.tw93.fun/posts/184-mamala

**Comment by ustcxmwu** [comment]
Andrej Karpathy 大神的这个「让我们来搞一个 GPT-2」很值得跟着学习一遍，124 分钟的长视频，受限构建 GPT-2 网络，然后优化对应训练速度，并按照 GPT-2 和 GPT-3 论文参数设置训练运行，挺适合周末时间我们来照着视频学习一下。

受限 -> 首先

**Reply by tw93** [reply]
谢谢指出

**Comment by rennsax** [comment]
Bartendar 我一直很喜欢的一个功能是能够把图标的间距缩小，不然我的常驻图标就没几个了😂 如果 ice 后面发展得好，应该会考虑切换，毕竟支持 FOSS。
